### DISCUSSION TITLE
add custom attribute in script tag

### DISCUSSION THREAD

**Discussion by artmedia** [oqtane/oqtane.framework#2266] [General] [answered]
Hello,
I'm trying to add this script in HEAD 
< script defer data-domain="artmedia.it" src="https://plausible.io/js/plausible.js" >< / script >

For all other scripts I typically use:  new Resource { ResourceType = ResourceType.Script, Url = "https://plausible.io/js/plausible.js"}
but in this case I have to add custom attributes "defer" and "data-domain".
How can I add these attributes?
Thanks in advance
Luca

**Comment by leigh-pointer** [comment]
Hi @artmedia could you not do something like this https://stackoverflow.com/questions/950087/how-do-i-include-a-javascript-file-in-another-javascript-file and use the new Resource { ResourceType = ResourceType.Scrip.... to reference the parent js file.

**Comment by artmedia** [comment, marked as answer]
@leigh-pointer,
great suggestion, I solved using jquery "appendTo" function placed in Theme.js of my Theme

$(document).ready(function () {
    $('< script>')
        .attr('defer','')
        .attr('data-domain', 'artmedia.it')
        .attr('src', 'https://plausible.io/js/plausible.js')
        .appendTo('head');
});
thanks

**Reply by leigh-pointer** [reply]
Pleased you found a solution.
